data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -93,6 +93,28 @@ def _combine_island_areas_with_states_and_set_thresholds(
 
         return df, threshold_column_name
 
+    def _create_low_income_threshold(self, df: pd.DataFrame) -> pd.Series:
+        """
+        Returns a pandas series (really a numpy array)
+        of booleans based on the condition of the FPL at 200%
+        is at or more than some established threshold
+        """
+        return (
+            df[
+                field_names.POVERTY_LESS_THAN_200_FPL_FIELD
+                + field_names.PERCENTILE_FIELD_SUFFIX
+            ]
+            >= self.LOW_INCOME_THRESHOLD
+        )
+
+    def _increment_total_eligibility_exceeded(
+        self, columns_for_subset: list, df: pd.DataFrame
+    ) -> None:
+        """
+        Increments the total eligible factors for a given tract
+        """
+        df[field_names.THRESHOLD_COUNT] += df[columns_for_subset].sum(axis=1)
```

Comment:
Looks like you forgot to return `df` here.

Reply:
Forgot to push (smh).

Reply:
That is not what I had in mind here. We are modifying an instance variable's reference in place. If I return that as it is being called then that would not be compiled properly.

You can check that by running `black .` with and without the return statement.

Reply:
Hmm. I'm not 100% sure I know what you mean by this. If you want to modify `self.df` without passing it back and forth, maybe just drop the `df` argument in the method then. So the method would look like: 

```
def _increment_total_eligibility_exceeded(
        self, columns_for_subset: list
    ) -> None:
        """
        Increments the total eligible factors for a given tract
        """
        self.df[field_names.THRESHOLD_COUNT] += self.df[columns_for_subset].sum(axis=1)
```

Reply:
Yeah, I could do that as well. Maybe that's more clearer anyway.